reth/reth-entrypoint

```diff
@@ -6,6 +6,7 @@ RPC_PORT="${RPC_PORT:-8545}"
 WS_PORT="${WS_PORT:-8546}"
 AUTHRPC_PORT="${AUTHRPC_PORT:-8551}"
 METRICS_PORT="${METRICS_PORT:-6060}"
+ADDITIONAL_ARGS=""
```

Comment:
I couldn't find the support for the `--override.fjord` flag on RETH. Let's dig deep further on this.

Reply:
I checked and found out that these property is already set in `reth`. 
Ref PR: https://github.com/paradigmxyz/reth/pull/8645

Reply:
Then, we'll need to update the reth version, right? It's not supported on reth v1.0.0 that we've set in the reth/Dockerfile.